Microsoft.Toolkit.Uwp.Services/project.json

```diff
@@ -1,5 +1,7 @@
 ﻿{
   "dependencies": {
+    "Microsoft.Graph": "1.0.1",
+    "Microsoft.IdentityModel.Clients.ActiveDirectory": "2.19.208020213",
```

Comment:
I think the muspec need to be changed as well:
https://github.com/Microsoft/UWPCommunityToolkit/blob/dev/build/Microsoft.Toolkit.Uwp.Services.nuspec#L21